stm/stm.ml

```diff
@@ -2473,8 +2477,12 @@ let known_state ~doc ?(redefine_qed=false) ~cache id =
                           ~drop_pt exn_info block_stop, ref false
                     in
                     qed.fproof <- Some (fp, cancel);
+                    let opaque = match keep with
+                      | VtKeep opaque -> opaque
+                      | _ -> Proof_global.Opaque (* this seems strange *)
```

Comment:
I don't know if using Opaque here makes sense. This branch is triggered by undo020 in the test suite, and probably by interactive use in coqide.

Reply:
If it is not keep, then it is drop (eg Admitted). So I guess opaque is fine